assets/src/application/ui/input/PriceTypeSign/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import CurrencySign from './CurrencySign';
+import { getCurrencySignCharacterCountClassName, getCurrencySignPositionClassName } from './utils';
+import PercentSign from './PercentSign';
+import { PriceType } from '@edtrServices/apollo/types';
+import useConfig from '@appServices/config/useConfig';
+
+type Props = {
+	priceType: PriceType;
+};
+
+const PriceTypeSign: React.FC<Props> = ({ priceType }) => {
+	const config = useConfig();
+	const sign = config?.currency?.sign;
+	const signB4 = config?.currency?.signB4;
```

Comment:
optional chaining is just for the sanity check

Reply:
interesting